activefires_pp/post_processing.py

```diff
@@ -398,21 +404,26 @@ def __init__(self, configfile, shp_borders, shp_mask, regional_filtermask=None):
         self.loop = False
         self._setup_and_start_communication()
 
-    def set_output_filename_parsers(self):
-        """Set the geojson output filename parsers."""
-        for output in self.national_outputs:
-            for prod in output:
-                output[prod].update({'parser': Parser(output[prod]['file_pattern'])})
-        for output in self.regional_outputs:
-            for prod in output:
-                output[prod].update({'parser': Parser(output[prod]['file_pattern'])})
+    # def set_output_filename_parsers(self):
+    #     """Set the geojson output filename parsers."""
+    #     for output in self.national_outputs:
+    #         for prod in output:
+    #             output[prod].update({'parser': Parser(output[prod]['file_pattern'])})
+    #     for output in self.regional_outputs:
+    #         for prod in output:
+    #             output[prod].update({'parser': Parser(output[prod]['file_pattern'])})
 
     def _setup_and_start_communication(self):
         """Set up the Posttroll communication and start the publisher."""
         logger.debug("Starting up... Input topic: %s", self.input_topic)
         now = datetime_utc2local(datetime.now(), self.timezone)
         logger.debug("Output times for timezone: {zone} Now = {time}".format(zone=str(self.timezone), time=now))
 
+        tic = time.time()
+        units = {'temperature': 'degC'}
+        self.unit_converter = UnitConverter(units)
+        logger.debug("Unit conversion initialization with Pint took %f seconds.", tic - time.time())
```

Comment:
Is this really needed?

Reply:
I wanted to track time consumption as I saw that Pint surprisingly to me required some seconds at each initialisation.